its/projects/ReproAzureFunctions/ReproAzureFunctions/Function1.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.Extensions.Logging;
+
+namespace ReproAzureFunctions
+{
+    public class Function1
+    {
+        private readonly ILogger _logger;
+
+        public Function1(ILoggerFactory loggerFactory)
+        {
+            _logger = loggerFactory.CreateLogger<Function1>();
+        }
+
+        [Function("Function1")]
+        public HttpResponseData Run([HttpTrigger(AuthorizationLevel.Function, "get", "post")] HttpRequestData req)
+        {
+            _logger.LogInformation("C# HTTP trigger function processed a request.");
+
+            var response = req.CreateResponse(HttpStatusCode.OK);
+            if (response == null)
+            {
+                response.WriteString("boom");
+            }
+            response.Headers.Add("Content-Type", "text/plain; charset=utf-8");
+
+            response.WriteString("Welcome to Azure Functions!");
+
+            return response;
+        }
```

Comment:
I'd prefer to reduce the scaffolding to something minimal, as it isn't needed to reproduce the problem.
```suggestion
        [Function("Function1")]
        public void Run([)
        {
            // FIXME: This should raise S1134
        }
```